linux_os/guide/system/permissions/restrictions/sysctl_net_core_bpf_jit_harden/rule.yml

```diff
@@ -0,0 +1,25 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Harden the operation of the BPF just-in-time compiler'
+
+description: '{{{ describe_sysctl_option_value(sysctl="net.core.bpf_jit_harden", value="2") }}}'
+
+rationale: |-
+    When hardened, the extended Berkeley Packet Filter just-in-time compiler
+    will randomize any kernel addresses in the BFP programs and maps,
```

Comment:
BFP should be BPF

Reply:
Fixed in 5de079de06a58e6009f477ca19f4b86038f0634f, thanks for spotting it.